packages/varlet-ui/src/bottom-navigation-item/BottomNavigationItem.vue

```diff
@@ -65,33 +56,27 @@ export default defineComponent({
     bindBottomNavigation(bottomNavigationItemProvider)
 
     watch(
-      () => badge.value,
+      () => props.badge,
       (newValue) => {
-        badgeProps.value = newValue === true ? defaultBadgeProps : badge.value
+        badgeProps.value = newValue === true ? defaultBadgeProps : props.badge
       },
       { immediate: true }
     )
```

Comment:
Why not got the badgeProps.value in computed

Reply:
Good idea,  I will replace `watch` with `computed`